src/wallet/wallet2.h

```diff
@@ -1297,6 +1297,8 @@ namespace tools
     void set_rpc_client_secret_key(const crypto::secret_key &key) { m_rpc_client_secret_key = key; m_node_rpc_proxy.set_client_secret_key(key); }
     uint64_t credits_target() const { return m_credits_target; }
     void credits_target(uint64_t threshold) { m_credits_target = threshold; }
+    bool enable_multisig() const { return m_enable_multisig; }
```

Comment:
```suggestion
    bool multisig_is_enabled() const { return m_enable_multisig; }
```
Can you disambiguate the two functions here? One `enables` and the other asks `is it enabled?`.